lux/extensions/auth/forms.py

```diff
@@ -31,20 +30,13 @@ def clean(self):
 class GroupForm(forms.Form):
     model = 'group'
     id = forms.HiddenField(required=False)
-    name = forms.CharField()
+    name = forms.CharField(validator=odm.UniqueField())
```

Comment:
I would use [SlugField](https://github.com/quantmind/lux/blob/master/lux/forms/fields.py#L405) for the name